distribution/docs/src/main/resources/content/_transformers/tika-input-xformer.adoc

```diff
@@ -7,11 +7,21 @@
 
 The Tika Input Transformer is the default input transformer responsible for translating Microsoft Word, Microsoft Excel, Microsoft PowerPoint, OpenOffice Writer, and PDF documents into Catalog Metacards.
 This input transformer utilizes https://tika.apache.org[Apache Tika] to provide basic support for these mime types.
-As such, the metadata extracted from these types of documents is the metadata that is common across all of these document types, e.g., creation date, author, last modified date, etc.
+As such, the metadata extracted from these types of documents is the metadata that is common across all these document types, e.g., creation date, author, last modified date, etc.
```

Comment:
change to:
"The metadata common to all of these document types, e.g., creation date, author, last modified date, etc., is extracted and used to create the catalog record."

Reply:
updated